src/App.js

```diff
@@ -1,16 +1,33 @@
-import React, { useState } from "react"
-import "./App.css"
-import LightSwitch from "./components/LightSwitch"
+import React, { useState } from "react";
+import "./App.css";
+import LightSwitch from "./components/LightSwitch";
+
 
 const App = () => {
-  const [lightSwitchStatus, setLightSwitchStatus] = useState("off")
 
   return (
-    <>
-      <h1>Lightbulb Challenge</h1>
-      <LightSwitch />
-    </>
-  )
+//Render a div element with a class name container and div elements inside it with classname box
+
+  <div className="container">
+  <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>  
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+    <div className="box">
+    <LightSwitch></LightSwitch></div>
+  </div>   
```

Comment:
See my comment above. While this block does work, you can use the component you created in LightSwitch.js to call the function as many times as you'd like.

Reply:
Yes, it does seem to work like that. Thanks for the feedback.